packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1543,8 +1539,6 @@ abstract class FlutterCommand extends Command<void> {
         if (_usesFatalWarnings) {
           globals.logger.fatalWarnings = boolArg(FlutterOptions.kFatalWarnings);
         }
-        // Prints the welcome message if needed.
-        globals.flutterUsage.printWelcome();
```

Comment:
Isn't this still needed?

Reply:
We print a consent message upon exit

https://github.com/flutter/flutter/blob/c783ce234459dea8924eaf48167811e8a6bd3d86/packages/flutter_tools/lib/src/base/process.dart#L647

Unless you are saying we should keep this until we actually remove the package completely